includes/Registry.php

```diff
@@ -70,6 +70,20 @@ public function get_post_to_post_relationship_by_key( $key ) {
 			return $this->post_post_relationships[ $key ];
 		}
 
+		// Fuzzy match the relationship by parsing the key and looping through the relationships
```

Comment:
Truthfully, I'm not in love with the fuzzy match. I think this could break down if a post type name contained the name of another post type. "post" and "postcard".

The challenge here is that the relationship name needs to be able to be constructed, however when the post type is one of the "many" set it does not have knowledge of it's siblings and the key can not be constructed properly.